iOS/Models/Attachments/PhotoAttachment.swift

```diff
@@ -34,9 +31,28 @@ struct PhotoAttachment: MediaItem {
     }
 
     var data: Data? {
-        return self._data
+        didSet {
+            self.convertDataIntoImage()
+        }
     }
 
-    var _data: Data?
     var info: [AnyHashable: Any]?
+
+    init(url: URL?, previewUrl: URL?, data: Data?, info: [AnyHashable : Any]?) {
+        self.url = url
+        self.previewUrl = previewUrl
+        self.data = data
+        self.info  = info
+
+        self.convertDataIntoImage()
```

Comment:
The photo attachment would make a new image every time the image was queried. Each image was a different instance so the displayable image view thought they were different.

This helps fix that issue so we don't load the same image multiple times.